### ISSUE TITLE
Missing `ember-native-dom-helpers` dependency for `drag-drop` helper

### ISSUE THREAD

**Issue by panthony** [mharris717/ember-drag-drop#129]
I'm using the `drag-drop` test helpers in my app and it fails with a missing dependency on `ember-native-dom-helpers`.

I believe we do not really want to list `ember-native-dom-helpers` in `dependencies` because we do not want this addon to be packaged in the dist file but maybe this dependency should be listed as required in README.md?

**Comment by nruth** [comment]
I've also hit this. It allows a test build, but the tests fail at runtime. 

```
Uncaught Error: 
Error: Could not find module `ember-native-dom-helpers` imported from `yourapp/tests/helpers/drag-drop`
```

Also is the dependency still needed now that https://github.com/cibernox/ember-native-dom-helpers ember-qunit provides similar things? There's a banner warning on the repo. 

![image](https://user-images.githubusercontent.com/26158/63230679-03282400-c208-11e9-8aff-531ef559ca8b.png)

**Comment by ondrejsevcik** [comment]
Same here. The PR #151 fixes this issue.

**Comment by dgavey** [comment]
This has been merged, should be fixed now  in 0.7.0 I hope.
